### PR TITLE
Update gitignore and add standard .npmignore used by plugins

### PR DESCRIPTION
**Pull request by platedbay**
I think I broke the package with the wrong change to the gitignore, my apologies! I basically stole this configuration from the other Gastby plugins. I also bumped up the version to fix this.

### CODE REVIEW

**Comment by platedbay** [comment]
Sorry for the spam, just wanted to cc ya @diegolamanno . Gatsby just released v2 so we're making some moves on our end, would love to have to not `link` this for development :)

File: .gitignore [hunk 1]
```diff
@@ -61,4 +61,7 @@ typings/
 .idea/
 .idea
 
-*.js
\ No newline at end of file
+/*.js
+!index.js
```
Comment (diegolamanno) [inline .gitignore]: We don't generate an `index.js` file. Why adding this line? The entry point of the plugin is: `"main": "gatsby-node.js",`

Comment (platedbay) [reply .gitignore]: Ah good catch! I think that was leftover from before. I'll remove 👍

**Comment by diegolamanno** [comment]
Hi, @platedbay I didn't get the notifications about your PR. We are already transitioning to V2. That includes updating this plugin. I will make a release at some point soon with another major release. We can leave `2.x` for Gatsby V1.

**Comment by platedbay** [comment]
All good @diegolamanno , thanks for jumping back onto this! Updated the .gitignore per your callout.

**Pull request merged**